op-program/chainconfig/chaincfg.go

```diff
@@ -104,12 +104,40 @@ func chainConfigByChainID(chainID eth.ChainID, customChainFS embed.FS) (*params.
 	return genesis.Config, nil
 }
 
-func mustLoadChainConfig(name string) *params.ChainConfig {
+func L1ChainConfigByChainID(chainID eth.ChainID) (*params.ChainConfig, error) {
```

Comment:
wondering if we want to move this somewhere under `op-service/eth` or so for reusability, so we don't need to reimplement this switch multiple times?